src/m365/todo/commands/task/task-add.ts

```diff
@@ -102,6 +121,32 @@ class TodoTaskAddCommand extends GraphCommand {
           return `'${args.options.reminderDateTime}' is not a valid ISO date string`;
         }
 
+        const regexCategory = new RegExp('^(?:None|Preset(?:[01]?[0-9]|2[0-4]))$');
+        if (args.options.categories) {
+          let testCategoriesResult = true;
+          args.options.categories.split(',').map(category => {
+            if (testCategoriesResult) {
+              testCategoriesResult = regexCategory.test(category);
+            }
+          });
+
+          if (!testCategoriesResult) {
+            return `categories can only containe the option 'None' or 'Preset0' to 'Preset24'`;
+          }
```

Comment:
I'm not quite sure this is the case. When I did a few tests, you needed to pass the display name of the category and not `preset0`. Have a quick look at the remarks section in the issue. That explains it a bit better.

Reply:
You're right! Even more, the api accepts any sort of string.. So ok for you to remore the validation here completely regarding the `categories` option?

Reply:
Yes indeed, I also don't think we need validation here as we can pass any string. But we should definitely include a section under remarks that talks a bit more about this option. Mainly about what users can insert here and how to work with the outlook categories